src/extended-events.js

```diff
@@ -86,6 +86,7 @@ class ExtendedEvents extends EventEmitter {
 
         if (Math.abs(_seekStart - _seekEnd) > 1) {
           _seeking = true;
+          resetPerVideoState();
```

Comment:
@e1adn looks good. I'd add a comment to say why it's for

Reply:
@tsi done